.github/workflows/test.yml

```diff
@@ -69,6 +69,13 @@ jobs:
         run: |
             [System.IO.File]::WriteAllLines((Join-Path -Path $env:USERPROFILE -ChildPath '.bazelrc'), 'startup --output_user_root=C:/tmp')
             [System.IO.File]::WriteAllLines((Join-Path -Path $env:USERPROFILE -ChildPath '.bazelrc'), 'build:windows --config=windows_no_dbg')
+      - name: Configure download mirrors
+        run: |
+            if [ ! -z "${{ secrets.WORKERS_MIRROR_URL }}" ] ; then
+              # Strip comment in front of WORKERS_MIRROR_URL, then substitute secret to use it.
+              sed -e '/WORKERS_MIRROR_URL/ { s@# *@@; s@WORKERS_MIRROR_URL@${{ secrets.WORKERS_MIRROR_URL }}@ }' -i WORKSPACE
+            fi
+        shell: bash
```

Comment:
`shell: bash` is present here but not for the release build, should it be there too?

Reply:
Thanks, fixing in the next push. :+1: